.goreleaser.yaml

```
@@ -0,0 +1,75 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines below are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+version: 2
+
+before:
+  hooks:
+    - cd local-interchain
+    - go mod tidy
+
+builds:
+  - id: "local-ic"
+    main: "./local-interchain/cmd/local-ic"
+    binary: "local-ic"
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -X main.Version=$(VERSION)
```

for spawn I use 

```
- -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser
```
so I think this needs to be

```suggestion
      - -X main.Version={{.Version}}
```

---

Good catch, this is exactly what the docs say as well. Not sure how i missed this given i was just working on this